.gitignore

```diff
@@ -15,3 +15,6 @@ dataOverlay/
 **/*.yml
 
 *.swp
+
+api/
+ftp/
```

Comment:
I have no problems adding these here, but I'm a bit confused.. why would these folders be here in `data-generation`..?

Reply:
I think it's coming from [these lines](https://github.com/MetabolicAtlas/MetabolicAtlas/blob/f780f4b597bbf01c1721577c94a26802152fdb17/proj.sh#L11-L12)

Reply:
Those two lines should be copying the contents of the `neo4j` and folder and the `dataOverlay` folder from this repo to the `neo4j/import` and `api` folders of the [MetabolicAtlas repo](https://github.com/MetabolicAtlas/MetabolicAtlas).

I'm not clear on the problem here either and I haven't encountered any similar issues with this repo before. It would be nice if you can elaborate on the problem here @mihai-sysbio.

Reply:
Sorry @MalinAhlberg for not paying more attention when sharing the previous link! It's obviously not related, but I looked only for the `$DATA_GENERATOR_PATH/` variable. @e0 pointed me in the right direction to what the real problem is.

These folders show up when [`generate-stack`](https://github.com/MetabolicAtlas/MetabolicAtlas/blob/f780f4b597bbf01c1721577c94a26802152fdb17/proj.sh#L16-L17) is _accidentally_ run in the `data-generation` folder instead of the core project folder. So gitignoring these will not really solve the problem, which is why I would close the PR and remove the commit.